app/helpers/access_request_helper.rb

```diff
@@ -4,6 +4,10 @@ def display_access_request_link?
     AccessRequestsController.feature_enabled? && current_user && !current_user.super_admin?
   end
 
+  def access_request_alternative_instruction
+    ENV['ACCESS_REQUEST_ALTERNATIVE_INSTRUCTION']
```

Comment:
just inline this ? `<% if text = ENV['ACCESS_REQUEST_ALTERNATIVE_INSTRUCTION'] %>`